### DISCUSSION TITLE
Next.js + Tailwind starter doesn't seem to work

### DISCUSSION THREAD

**Discussion by corysimmons** [codesandbox/codesandbox-client#8417] [Q&A]
What am I doing wrong here?

https://codesandbox.io/p/devbox/green-surf-282ltp?file=%2Fapp%2Fpage.tsx%3A10%2C1

![image](https://github.com/codesandbox/codesandbox-client/assets/224636/482f6bcf-23ea-46a2-a9f2-ef0bb1c67378)

**Comment by filipelima18** [comment]
Hey @corysimmons! It looks like this Devbox is private - would it be possible at all to temporarily change its privacy to "Unlisted" so we can investigate further?

**Reply by corysimmons** [reply]
Sorry, I didn't realize it was private. It's public now.
When it initially loaded it seemed to work, but if you refresh the box it throws those hydration errors.

**Reply by filipelima18** [reply]
After [forking](https://codesandbox.io/p/devbox/green-surf-forked-jfhqdx?file=%2F.devcontainer%2Fdevcontainer.json) this Devbox and restarting the VM, I don't see the hydration errors any longer. Can you check if restarting the Devbox solves it for you? You can do it from the menu on the top left as shown below.

![brave_5AUi4VyCis](https://github.com/codesandbox/codesandbox-client/assets/45261568/9cb4bd7e-3adb-4ac0-9977-9525689a1c8f)

**Reply by corysimmons** [reply]
I tried, but that didn't fix it for me.

**Reply by CompuIves** [reply]
I wonder, can you repro the same issue in other browsers? I'm curious if maybe a browser extension is manipulating the HTML, causing this error. I cannot reproduce the same problem right now.

**Reply by corysimmons** [reply]
@CompuIves Sorry for the slow response.

- It does work fine in Firefox.
- It does this hydration error in Incognito with no extensions.
- I get the hydration error on fresh copy of Edge.

It seems to be a Chromium bug in general, but it's not affecting you? 🤔

**Comment by serranoarevalo** [comment]
Unrelated but https://codesandbox.io/p/devbox/green-surf-282ltp?file=%2Fapp%2Ftailwind.css%3A1%2C1-3%2C23 contains:

```css
@tailwind 'base';
@tailwind 'components';
@tailwind 'utilities';
```

And it should be:

```css
@tailwind base;
@tailwind components;
@tailwind utilities;
```

**Reply by corysimmons** [reply]
Ah good catch. I've updated it, but, as you said, the hydration error is still present.
